test/tools.ts

```diff
@@ -88,6 +100,12 @@ export function resolveDutType() {
 
 export function createDeviceInteractor(testbotHat: TestBotHat) {
 	const dutType = resolveDutType();
+	if (dutType === 'fincm3') {
+		if (process.env.BALENA_FIN_V09 === 'true') {
+			return new BalenaFinV09(testbotHat);
+		}
+		return new BalenaFin(testbotHat);
+	}
```

Comment:
Would it be better to implement a switch here for future scalability?

Reply:
@vipulgupta2048 so I was thinking about this but I don't want to add complexity for multiple device types for the balenaFin as from a 'balena' perspective there only exists one balenaFin device type. So without adding complexity to splitting the balenaFin into having 2 seperate devices types I'm not certain how we maintain this?

Reply:
I remember we talked about this. Since currently, we don't have a solution for both Fin deviceInteractor's to be in the same class. We can keep it like this and figure out a way to merge both of them to match the balena perspective. 

As more devices get supported on testbot. We need to make sure the code is still readable. My suggestion is to use a [switch](https://www.typescripttutorial.net/typescript-tutorial/typescript-switch-case/) instead of 2 IF statements or even use `else if`.